tasks/system.yml

```diff
@@ -16,15 +16,12 @@
       backup: yes
     notify: "systemd-journald restart"
 
-# - name: Nuke journald console output
-#   replace:
-#     path: /etc/systemd/journald.conf
-#     regexp: '^#*TTYPath=.*'
-#     replace: 'TTYPath=/dev/null'
-#     backup: yes
-#   notify: "systemd-journald restart"
-
-  - name: Install aptitude, dnsutils, jq, rclone, vim
+  - name: Install aptitude, dnsutils, jq, vim
     apt:
-      name: [ aptitude, dnsutils, jq, rclone, vim ]
+      name: [ aptitude, dnsutils, jq, vim ]
       update_cache: yes
+
+  - name: Install latest rclone
+    shell:
+      cmd: "curl https://rclone.org/install.sh | sudo bash"
```

Comment:
I would rather download a deb and install that rather that pipe a command into sudo bash

Unfortunately, rclone doesn't have a deb repo, and I don't think there is a link to a "latest" deb from github; you can only download release by version.

Reply:
https://github.com/rclone/rclone/releases/latest

Reply:
found a deb package on latest release.